src/auth.nim

```diff
@@ -158,6 +164,27 @@ proc release*(account: GuestAccount) =
   dec account.pending
 
 proc getGuestAccount*(api: Api): Future[GuestAccount] {.async.} =
+  let now = epochTime().int
```

Comment:
See the comment above for a different way of doing this

Reply:
done